### DISCUSSION TITLE
How to setup Statistics with Encoders Like RadioBoss / RadioCaster?

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#4061] [Q&A]
Hey everyone!

I've recently started using AzuraCast and it's worked great I'm only the issue though of statistics not loading up, the station is setup in RadioCaster streams and plays awesome. 

I've got 3 unique listeners according to AzuraCast but encoder says 0 

When I choose get stats from server it'll change to azuragetcast or something like that don't have the pc infront of me at the moment and stay at 0,

When I choose to manually do it and use same info I used for the encoder it'll say HTTP cant find or link not found.

Happens in Icecast / Shoutcast any ideas on how to get the statistics to load on encoders? 

Any insight would be appreciated it.

**Comment by WSradio** [comment]
For RadioBoss: 
1. Go to the encoder screen for your stream.
2. Click Statistics tab
3. Click on specify the server to get statistics from
4. select you server type (ie: Icecast)
5. Server should be the LISTENING URL and port (not the one you stream TO) 
6. for password put admin:xxxxxxxx  where xxxxxx is the admin password listed on the azuracast profile page
7. set stream ID as the listening mount point (ie: /live.mp3)
8. Click ok and it should work.
